File: Fledge/FledgeJava/app/src/main/java/com/example/adservices/samples/fledge/ServerAuctionHelpers/SelectAdsResponse.java
/*
 * Copyright (C) 2023 The Android Open Source Project
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
package com.example.adservices.samples.fledge.ServerAuctionHelpers;
/**
 * A simple SelectAdResponse POJO used to convert to and from json that will be returned as a
 * response by HTTPS POST SelectAd call.
 *
 * <p>https://github.com/privacysandbox/fledge-docs/blob/main/bidding_auction_services_api.md#sellerfrontend-service-and-api-endpoints
 */
public class SelectAdsResponse {
  public String auctionResultCiphertext;
}

File: Fledge/FledgeKotlin/app/src/main/java/com/example/adservices/samples/fledge/serverauctionhelpers/SelectAdsResponse.kt
package com.example.adservices.samples.fledge.serverauctionhelpers

/**
 * A simple SelectAdResponse POJO used to convert to and from json that will be returned as a
 * response by HTTPS POST SelectAd call.
 *
 *
 * https://github.com/privacysandbox/fledge-docs/blob/main/bidding_auction_services_api.md#sellerfrontend-service-and-api-endpoints
 */
class SelectAdsResponse {
    var auctionResultCiphertext: String? = null
}